src/applications/disability-benefits/526EZ/pages/veteranInfo.js

```diff
@@ -1,64 +1,31 @@
-import { merge } from 'lodash/fp';
+import AsyncDisplayWidget from '../components/AsyncDisplayWidget';
 
-import fullSchema526EZ from 'vets-json-schema/dist/21-526EZ-schema.json';
-
-import currentOrPastDateUI from '../../../common/schemaform/definitions/currentOrPastDate';
-import fullNameUI from '../../../common/schemaform/definitions/fullName';
-import ssnUI from '../../../common/schemaform/definitions/ssn';
-import { genderLabels } from '../../../../platform/static-data/labels';
-
-import ReviewCardField from '../components/ReviewCardField';
-
-import {
-  veteranInformationViewField,
-  VAFileNumberDescription
-} from '../helpers';
+import { veteranInformationViewField } from '../helpers';
 
 export const uiSchema = {
-  'ui:field': ReviewCardField,
+  'ui:field': 'StringField',
+  'ui:widget': AsyncDisplayWidget,
   'ui:title': 'Veteran information',
   'ui:options': {
-    viewComponent: veteranInformationViewField
-  },
-  fullName: fullNameUI,
-  socialSecurityNumber: merge(ssnUI, {
-    'ui:title': 'Social Security number (must have this or a VA file number)',
-    'ui:required': form => !form.vaFileNumber
-  }),
-  vaFileNumber: {
-    'ui:title': 'VA file number (must have this or a Social Security number)',
-    'ui:required': form => !form.socialSecurityNumber,
-    'ui:help': VAFileNumberDescription,
-    'ui:errorMessages': {
-      pattern: 'Your VA file number must be between 7 to 9 digits'
-    }
-  },
-  dateOfBirth: currentOrPastDateUI('Date of birth'),
-  gender: {
-    'ui:title': 'Gender',
-    'ui:options': {
-      labels: genderLabels
+    viewComponent: veteranInformationViewField,
+    loadingMessage: 'Fetching your information...',
```

Comment:
@peggygannon will probably recommend that we use "Loading your information" or something else here as I don't recall using "Fetch" in these messages before.